config/routes.rb

```diff
@@ -661,7 +661,9 @@
 
     resources :badges, only: [:index]
     get "/badges/:id(/:slug)" => "badges#show", constraints: { format: /(json|html|rss)/ }
-    resources :user_badges, only: [:index, :create, :destroy]
+    resources :user_badges, only: [:index, :create, :destroy] do
+      put "favorite" => "user_badges#favorite", constraints: { format: :json }
```

Comment:
I would really go with toggle_favorite if possible

Reply:
SGTM - done!

Reply:
@iamricard I'm not seeing the changes, did you forgot to push them?

Reply:
I did 🤦🏼 I pushed it now!